### PR TITLE
Add commit history bar graph: fix #89

### LINKED ISSUE #89: Add commit history bar graph
GitHub has a nice bar graph showing the Ruffle repo's commit history: https://github.com/ruffle-rs/ruffle/graphs/commit-activity
image
In my opinion, this bar graph more accurately reflects the project progress and activity than the AVM1 and AVM2 progress bars. It shows that things are consistently being accomplished every week as code is committed.
I'd like to see a version of this bar graph along the bottom of the "Compatibility" section of the Ruffle website. Hopefully it can be nicely styled, similarly to the progress bars (but the bars would be vertical, not horizontal).
The data can be requested straight from GitHub's API using this URL: https://api.github.com/repos/ruffle-rs/ruffle/stats/commit_activity
In that data, the "week" value is a Unix epoch giving the start date of the week, and "total" is the number of commits during that week.

### PR DESCRIPTION
**Pull request by danielhjacobs**
Show that there is constant activity, even when compatibility is not updated. Uses [Chart.js](https://www.chartjs.org/) - Open source charts licensed under the MIT License.

Some info here may also be useful to include: https://jsfiddle.net/ndbatL8j/, but I'm not sure what.

Screenshot:
Screenshot from 2022-06-06 16-25-52

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -339,6 +339,11 @@ <h5>ActionScript 3 API <code>5%</code></h5>
 					</p>
 				</div>
 			</div>
+			<div class="row">
+				<div class="col-base col-lg d-flex justify-content-center">
+					<canvas id="weeklyCommits"></canvas>
```
Comment (relrelb) [inline index.html:344]: I think the naming convention is [`kebab-case`](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) - `weekly-commits`

File: _includes/footer.html [hunk 2]
```diff
@@ -23,6 +100,21 @@
         }
       });
     });
+    const labels = [];
+    const dat = [];
+    fetch("https://api.github.com/repos/ruffle-rs/ruffle/stats/commit_activity")
+      .then(response => response.json())
+      .then(data => {
+        for (let row of data) {
```
Comment (relrelb) [inline _includes/footer.html:108]: Can do `const row` here.

File: _includes/footer.html [hunk 3]
```diff
@@ -2,7 +2,84 @@
...
 <script type="text/javascript" src="{{ "/assets/bootstrap/js/bootstrap.min.js" | relative_url }}"></script>
+<script src="https://cdn.jsdelivr.net/npm/chart.js@3.7.1/dist/chart.min.js"></script>
 <script type="text/javascript">
+  function createChart(labels, dat) {
+    const ctx = $("#weeklyCommits");
+    const data = {
+      labels: labels,
+      datasets: [{
+        data: dat,
+        backgroundColor: "#FFAD33",
```
Comment (relrelb) [inline _includes/footer.html:13]: I think we should have CSS "Ruffle blue" and "Ruffle orange" color constants, like we have in Ruffle itself:
https://github.com/ruffle-rs/ruffle/blob/b8cdd598771927e7229cd633599771eb2161f5bd/web/packages/core/src/shadow-template.ts#L9-L10

File: _includes/footer.html [hunk 4]
```diff
@@ -2,7 +2,84 @@
...
+        },
+        tooltip: {
+          backgroundColor: "#FFAD33",
+          borderWidth: 1,
+          borderColor: "#1a2743",
+          titleColor: "#37528C",
+          titleMarginBottom: 0,
+          callbacks: {
+            title: (t) => {
+              return t[0].formattedValue + " commits the week of " + new Date(t[0].label).toLocaleString(undefined, {
```
Comment (relrelb) [inline _includes/footer.html:64]: Same as https://github.com/ruffle-rs/ruffle-rs.github.io/pull/90/files#r897128526.

File: _includes/footer.html [hunk 5]
```diff
@@ -2,7 +2,84 @@
...
+          },
+          grid: {
+            drawBorder: false,
+            display: false
+          }
+        },
+        x: {
+          ticks: {
+            maxTicksLimit: (context) => {
+              return context.chart.width > 500 ? 18 : 9;
```
[the remark this answers is not in the document]
Comment (danielhjacobs) [reply _includes/footer.html]: I opted for a full-blown arrow-function. It seemed more appropriate to be more terse when already trying to be terse by using the ternary operator.

File: _includes/footer.html (hunk 4 above)
Comment (danielhjacobs) [reply _includes/footer.html]: I opted for an arrow function here as well. This one I was more unsure about, as it's a long statement, but it is still just a single line (and I'd argue it should be), and arrow-functions are most appropriate when using just a single line.

**Review by relrelb — APPROVED** [review]

**Comment by relrelb** [comment]
Thank you!

**Pull request merged**